.circleci/config.yml

```diff
@@ -7,6 +7,17 @@ jobs:
     steps:
       - checkout
 
+      - run:
+          name: Run audit-ci
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v8.11.4
+            nvm alias default v8.11.4
+            npm install -g npm
+            npm install --no-audit
```

Comment:
Is this installing the dependencies? If so, it's kind of duplicating effort since a later step does a `yarn install`. But if we can only run the audit through npm, I'm not sure what we can do

Reply:
Ok.  Updated to use `yarn` for audit, and fixed one package reported by `yarn` audit.

Reply:
Image `node:8.11.4` has `yarn` version `1.6.0` and `>=1.12.3` for audit to work.  The `node:10.13.0` has `yarn 1.12.3`.

I got permission error if I update the yarn version in docker container.  `docker run --rm -v /home/circleci/.cache:/root/.cache -v $(pwd):/home --workdir /home node:8.11.4 /bin/bash -c  'npm install -g yarn && yarn audit-ci --cache-folder /root/.cache'`
`/bin/bash: /usr/local/bin/yarn: Permission denied`
So I still need to run audit-ci outside of the container.